dictionary/en.yaml

```diff
@@ -2147,11 +2162,21 @@ gza:
   family: "R"
   gloss: "agentive cause"
   short: "[E:tce* pan] is an agentive cause of [A:()] being true."
+cmuel:
+  id: k59p1rkb54
+  family: "R"
+  gloss: "school"
+  short: "[E:tce* pan] is a school."
 kini:
   id: macaegnams
   family: "R"
   gloss: "learns"
   short: "[E:tce* pan] learns/studies fact [A:()]."
+kinu:
+  id: z486re43l0
+  family: "R"
+  gloss: "teaches"
+  short: "[E:tce* pan] teaches fact [A:()]."
```

Comment:
We could merge it with `kini` as `E learns fact A from O`, allowing to express the student-teacher relation.